### PR TITLE
Upgrade guide updates for v3.5

### PR DESCRIPTION
**Pull request by esethna**
@jasonblais can you please help verify the SQL query upgrade note for this release? 

image

### CODE REVIEW

**Comment by jasonblais** [comment]
The SQL query upgrade issues are no longer applicable for 3.2 --> 3.5.

The upgrade from 3.2 --> 3.4 will still have the migration issue, but it's included in the changelog (see below). Hence, I think it's safe to remove those notes from this guide.

image

**Comment by esethna** [comment]
Thanks, updated

File: source/administration/upgrade.md [hunk 1]
```diff
@@ -21,11 +21,7 @@ To start, select one of the following guides:
 1. Download the **appropriate next upgrade** of your Team Edition server and note any compatibility procedures
       1. Run `platform -version` to check the current version of your Mattermost server
       2. Determine the appropriate next upgrade for your server:
-          - Mattermost `v3.0.x` and above can upgrade directly to Mattermost `v3.4.x`
-              - Note: If public links are enabled, upgrading to `v3.4.x` will invalidate existing public links due to a security upgrade allowing admins to invalidate links by resetting a public link salt from the System Console.
```
Comment (jasonblais) [inline source/administration/upgrade.md]: Public links become invalidated if you upgrade from v3.3 or earlier to v3.5 though

Comment (esethna) [inline source/administration/upgrade.md]: Thanks, updated

**Pull request merged**